ts41/get.d.ts

```diff
@@ -0,0 +1,131 @@
+import {Split} from './utilities';
+import {StringDigit} from '../source/utilities';
+
+/**
+Like @see {@link Get} but receives an array of strings as a path parameter.
```

Comment:
I don't think this is valid `@see` syntax. VS Code renders it raw and https://tsdoc.org/play only renders `Like`.

Reply:
Yeah, it should be at the beginning of the line: https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#jsdoc-see-tag